### DISCUSSION TITLE
How to re-run after npm publish failed

### DISCUSSION THREAD

**Discussion by bostrom** [semantic-release/semantic-release#1888] [Q&A] [answered]
I'm setting up semantic-release for the first time and have a bunch of changes in my beta branch, which is set up to publish a pre-release version to npm. Due to some 2FA trouble at npmjs.com, the first run of semantic-release on the beta branch was unsuccessful at the publish step. 

I corrected the token issues and re-ran the GH Action for the beta branch. I expected it to re-run the whole process and publish my beta version, set up release notes etc, but instead it "Found 0 commits since last release", so it quit after analyzing commits. 

How can I have semantic-release re-publish the beta version?

**Comment by travi** [comment, marked as answer]
you are facing a partial release because semantic-release was able to create a tag for the new version, but was unable to actually publish. this is known to happen in situations like you describe. you can manually remove the tag from the repository, which would then allow you to re-run with the results you originally expected

**Reply by bostrom** [reply]
Thanks for the explanation. That did the trick, indeed! 🙏
